app/components/drag-handle.js

```diff
@@ -51,32 +50,18 @@ export default Component.extend({
   'on-drag'() {},
 
   startDragging() {
-    let $container = this.$().parent();
-    let $containerOffsetLeft = $container.offset().left;
-    let $containerOffsetRight = $containerOffsetLeft + $container.width();
-    let namespace = `drag-${this.get('elementId')}`;
+    let namespace = `drag-${this.elementId}`;
 
     this.sendAction('action', true);
 
-    $('body').on(`mousemove.${namespace}`, e => {
-      let position = this.get('isLeft') ?
-        e.pageX - $containerOffsetLeft :
-        $containerOffsetRight - e.pageX;
-
-      position -= this.get('left');
-      if (position >= this.get('minWidth') && position <= this.get('maxWidth')) {
-        this.set('position', position);
-        this.get('on-drag')(position);
-      }
-    })
-      .on(`mouseup.${namespace} mouseleave.${namespace}`, () => {
-        this.stopDragging();
-      });
+    document.body.addEventListener(`mousemove.${namespace}`, this._mouseMoveHandler);
```

Comment:
These also need to be cleaned up in`willDestroy` or `willDestroyElement`

Reply:
`willDestroyElement` calls `this.stopDragging();` which has the `removeEventListener` calls. Did I mess them up?

Reply:
Ahh, I see, thank you for explaining!